src/flownet/config_parser/_config_parser.py

```diff
@@ -2333,3 +2317,45 @@ def _check_defined(path_in_config_dict: dict, parameter: str):
     param_dict.pop("distribution")
     param_dict.pop("low_optimistic", None)
     return {key for key, value in param_dict.items() if value is not None}
+
+
+def _check_if_all_region_priors_defined(
+    path_in_config_dict: dict, unique_regions: List, parameter_name: str
+):
+    """
+
+    Args:
+        path_in_config_dict: a location in the config schema dictionary
+        unique_regions: a list of the unique region numers in the input simulation model
+        parameter_name: the name of the output region parameter
+
+    Returns:
+        Nothing, raises ValueErrors if something is wrong
```

Comment:
```suggestion
        Nothing

    Raises:
        ValueError: if something is wrong.
```
"if something is wrong" _could_ be slightly more specific 😃

Reply:
Yes, definitely. Changed the docstring in some other functions as well. Some can potentially raise many different ValueErrors. Not entirely clear to me how that should be written in the docstring, but an attempt here, at least.